packages/mediacenter/kodi/patches/kodi-100.21-remove-libdvd-requirement.patch

```diff
@@ -0,0 +1,57 @@
+From 7cf46841663438ea3f470429fa31f19a002c71a5 Mon Sep 17 00:00:00 2001
```

Comment:
you dont need this patch if you build with -DENABLE_DVDCSS=ON (not sure if that has been fixed upstream yet, but LE users will want dvdcss anyway)